assets/model_monitoring/components/src/data_drift_compute_metrics/numerical_data_drift_metrics.py

```diff
@@ -95,40 +96,40 @@ def _jensen_shannon_numerical(
         production_df_count,
         numerical_columns,
     )
+
     baseline_histograms = get_histograms(baseline_df, bin_edges, numerical_columns)
     prod_histograms = get_histograms(production_df, bin_edges, numerical_columns)
 
-    baseline_freq_count_map = _calculate_column_value_frequency_bin_map_in_df(
-        baseline_histograms, numerical_columns, baseline_df_count
-    )
-    production_freq_count_map = _calculate_column_value_frequency_bin_map_in_df(
-        prod_histograms, numerical_columns, production_df_count
-    )
+    baseline_histograms_counts = {key: value[1] for key, value in baseline_histograms.items()}
+    prod_histograms_counts = {key: value[1] for key, value in prod_histograms.items()}
 
-    reference_distribution = list(
-        baseline_freq_count_map["feature_bucket_frequency_map"].values()
-    )
+    baseline_histograms_percent = {}
+    for key, values in baseline_histograms_counts.items():
+        for value in values:
+            baseline_histograms_percent[key] = value / baseline_df_count
 
-    transformed_production_df = _map_production_df_columns_with_bins(
-        production_df, production_freq_count_map
-    )
-    distributionBalanceMeasure = (
-        DistributionBalanceMeasure()
-        .setSensitiveCols(numerical_columns)
-        .setReferenceDistribution(reference_distribution)
-        .transform(transformed_production_df.select(numerical_columns))
-    )
+    prod_histograms_percent = {}
```

Comment:
Small bug here. lines 111 to 114 should read:
```python
    prod_histograms_percent = {}  
    for key, values in prod_histograms_counts.items():  
        value_list = []  
        for value in values:  
            value_list.append(value / production_df_count)  
        prod_histograms_percent[key] = value_list
```

Reply:
Updated.